src/utils/generateParamString.ts

```diff
@@ -0,0 +1,7 @@
+export const encodeToParamString = (string: string) => {
+  return string.replaceAll(' ', '-')
+}
+
+export const decodeParamString = (string: string) => {
```

Comment:
Once we remove `task.title` from our URL this should no longer be required

Reply:
I think having `task.title` in the route param is a good approach for the reasons mentions [here](https://github.com/copilot-platforms/tasks-app/pull/15#discussion_r1577753143). 

If there are any potential reasons for this to be removed then please do let me know thoughts.